app/code/Tasks/FeedbackManager/Controller/Manage/Edit.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Manage;
+
+use Magento\Customer\Controller\AbstractAccount;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Customer\Model\Session;
+
+class Edit extends \Magento\Framework\App\Action\Action
```

Comment:
@gayathri-aravind Pls fix deprecated methods and classes

Reply:
Replaced the deprecated class \Magento\Framework\App\Action\Action with \Magento\Framework\App\ActionInterface . Commit: 882571dc9e3988e3e1f109a2d06fe316213a2971